arangod/Aql/AqlFunctionsInternalCache.cpp

```diff
@@ -92,9 +92,20 @@ icu::RegexMatcher* RegexCache::buildSplitMatcher(AqlValue const& splitExpression
   return fromCache(rx, _likeCache);
 }
 
+arangodb::ValidatorBase* AqlFunctionsInternalCache::buildValidator(VPackSlice const& validatorParams) {
```

Comment:
I think it should be named `buildSchema`, because from the user perspective there are just schemas.

Reply:
Comment added -- While it only returns the Validator for `JSONSchema` at the moment the cache will work for any Validator that inherits from `ValidatorBase` without change.